packages/aragon-wrapper/src/index.js

```diff
@@ -1528,6 +1530,22 @@ export default class Aragon {
     let scriptData = script.substring(10)
     while (scriptData.length > 0) {
       const { segment, scriptLeft } = decodePathSegment(scriptData)
+      const { selector, data } = segment
+
+      if (selector === FORWARD_SIG) {
+        const forwardCallData = data.substring(10)
+        const forwardDataOffset = parseInt(`0x${forwardCallData.substring(0, 64)}`, 16) * 2
+        const forwardDataStartIndex = forwardDataOffset + 64
+        const forwardDataLength = parseInt(`0x${forwardCallData.substring(forwardDataOffset, forwardDataStartIndex)}`, 16) * 2
+        const forwardData = `0x${forwardCallData.substring(forwardDataStartIndex, forwardDataStartIndex + forwardDataLength)}`
+
+        // If forwarding data is a script decode it
+        const forwardScriptId = script.substring(0, 10)
+        if (forwardScriptId === CALLSCRIPT_ID) {
+          segment.children = this.decodeTransactionPath(forwardData)
+        }
+      }
```

Comment:
Let's add some comments here to explain the offsetting. Best to be explicit (I get confused all the time looking at offsets). Perhaps "munching" on each operation (i.e. consuming the first part of the string via `substring(x)`) would also make it more clear.

We may also want to add in a bit of error checking for the lengths in case someone puts in garbage data into an EVMScript.